app/Http/Controllers/BlogRevisionController.php

```diff
@@ -47,16 +47,20 @@ public function create(Request $request) {
             'title' => ['required', 'string'],
             'article_id'=> ['required', 'string', 'regex:/^[A-Za-z0-9_\-]+$/'],
             'content' => ['required', 'string'],
-            'handle_name' => ['string']
+            'handle_name' => ['string', 'nullable']
         ]);
 
+        $handle_name = $request->input('handle_name');
+        
+        if($handle_name == '')$handle_name = NULL;
```

Comment:
ごめん忘れてた ここ怖いから`===`使ってほしいです
```suggestion
        if($handle_name === '')$handle_name = NULL;
```